### PR TITLE
fix pins for 2.50.0

### PR DESCRIPTION
**Pull request by katietz**
* increase build number
* remove wrong ignore_run_exports
* enable fontconfig
* enable typefont

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -28,11 +28,6 @@ build:
...
     - '**/ld-linux-x86-64.so.2'  # [linux]
     - '**/ld64.so.1'             # [linux]
-  ignore_run_exports:
-    - pixman
-    - freetype
-    - fontconfig
-    - gdk-pixbuf
 
 requirements:
   build:
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:33]: I found `bison` only in [CMakeLists.txt](https://gitlab.com/graphviz/graphviz/-/blob/main/CMakeLists.txt) but we use cmake only on win. Do we really need it on unix?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -79,34 +130,35 @@ requirements:
...
     # Not linked in directly, likely interface deps? add 'em anyway!
     - gdk-pixbuf                                          # [unix]
-    - fontconfig                                          # [unix]
-    - freetype                                            # [unix]
+    - fontconfig
+    - freetype
+    - poppler
 
 test:
   files:
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:164]: `doc_source_url` can be removed

File: recipe/meta.yaml (hunk 1 above)
Comment (katietz) [reply recipe/meta.yaml:33]: yes, from my findings it seems to be used in some sub-code-paths

**Review by skupr-anaconda — APPROVED** [review]

File: recipe/meta.yaml (hunk 2 above)
Comment (katietz) [reply recipe/meta.yaml:164]: done, thanks

**Pull request merged**